### DISCUSSION TITLE
Upgrade Babylon template

### DISCUSSION THREAD

**Discussion by setasan** [primefaces/community#58] [PrimeReact Templates] [answered]
We are using Babylon application template and we need to upgrade it to last version. we also need typescript support. How can we proceed to get this done?

**Comment by atkntepe** [comment]
Hi, we're currently planning a new chapter for our Premium templates. We're introducing what we like to call 'next-gen' templates, which will feature full updates, new features, and complete remakes. At this moment, we haven't finalized which of our existing templates will make it to this next-gen list, but Babylon is one of the candidates. Stay tuned for updates.👍

**Comment by cagataycivici** [comment, marked as answer]
Babylon will be moved to Next.js and Typescript but needs a redesign first, it misses dark mode especially and lacks new menu modes. I can't give an exact timeframe yet though.

**Reply by ivanbrizuela** [reply]
Are there any updates on whether Babylon template will be upgradable to `primefaces 10`? We are stuck with this.
